azure-pipelines.yml

```diff
@@ -47,6 +66,13 @@ jobs:
       artifact: windows
       targetPath: bin
     condition: succeededOrFailed()
+  # NOTE: that PublishBuildArtifacts is required for signing
```

Comment:
There was a PR to change this, or at least support the other way. I think a property was added. But, this is not required.

Reply:
It failed when I used `PublishPipelineArtifact` in a weird way:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4392505&view=logs&j=e24aea9e-642d-565f-0ea4-ee7a5a3b8fdc&t=6e777e80-cc0f-5ccb-1c72-bf4d88543f78&l=12

But I'm fine to use `PublishBuildArtifacts`, it works. 👍

Reply:
Again I don't have a link on my phone, but I found the or and it was not merged yet. 🙃

Reply:
This one: https://github.com/xamarin/yaml-templates/pull/96

But I'm fine to leave this as it is.

Reply:
Yup, but yeah. No need to change. Just was an FYI for future things.